packages/ibm-products/src/components/BasicHeader/BasicHeader.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright IBM Corp. 2020, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { ComboButton } from '..';
+import { BasicHeader } from './BasicHeader';
+
+describe(ComboButton.name, () => {
```

Comment:
Should update this to `BasicHeader` so it outputs correctly in our tests. :)

I think most of the more recent tests follow this:

```jsx
const componentName = BasicHeader.displayName;
const blockClass = `${pkg.prefix}--basic-header`;

describe(componentName , () => {
  //
})
```

Reply:
one idea: It would be good to have snippets or commands to create placeholders for tests , components and scss files